docs/developers/fuse-box/fusebox-intro.mdx

```diff
@@ -20,19 +20,20 @@ The Fuse API is a REST API that Developers can use to access particular features
 - Trade API: This API facilitates trading activities and provides access to trading-related data.
 
 - **Smart Wallet API v2:** The Smart Wallet API v2 (previously referenced as Smart Wallets, now deprecated) enables Developers to Authenticate and access a `jwt` string which they can then parse to enable **sessions** for users. This enables users who Authenticate in the DApp provided by the Developer and are recognised upon sign out and sign in without the need to re-authenticate. The endpoints available are:
-  - `Auth`
   - `Create Wallet`
-  - `Get Wallet`
-  - `Get Historical Transaction`
+  - `Get Wallet Action`
 
 Below is an example using curl to create a Smart Contract Account using the endpoint
```

Comment:
This sentence should be "Below is an example using the SDK to create a Smart Contract Account"